src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -254,13 +359,43 @@ def test_opencl_kernel_code_generation():
         "  int j = get_global_id(1);\n"
         "  cu[j * cuLEN1 + i] = ((0.5e0 * (p[j * pLEN1 + (i + 1)]"
         " + p[j * pLEN1 + i])) * u[j * uLEN1 + i]);\n"
-        "}\n"
+        "}\n\n"
         )
 
     openclwriter = OpenCLWriter()
     assert expected_code == openclwriter(kschedule)
 
 
+def test_opencl_kernel_output_file(kernel_outputdir):
+    '''Check that an OpenCL file named modulename_kernelname_0 is generated.
+    '''
+    import os
+    psy, _ = get_invoke("single_invoke.f90", API, idx=0)
+    sched = psy.invokes.invoke_list[0].schedule
+    otrans = OCLTrans()
+    otrans.apply(sched)
+    sched.kernels()[0].name = "name"
+    generated_code = str(psy.gen).lower()
```

Comment:
pylint complains that `generated_code` is unused. We could just remove the assignment part of this statement altogether (as I think we've agreed/voted to not use `_` in tests unless absolutely necessary).

Reply:
This is needed because it generates the OpenCL kernel as side-effect. I removed the `str().lower()`, used `_` and added a comment mentioning the side-effect.

Reply:
Great, thanks.